<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatUi</class>
 <widget class="QWidget" name="ChatUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>253</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Chat      From: </string>
       </property>
       <property name="buddy">
        <cstring>chat_work_area_</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fromLE"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destLE"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendPB">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="lyx::frontend::EmbeddedWorkArea" name="chat_work_area_" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lyx::frontend::EmbeddedWorkArea</class>
   <extends>QWidget</extends>
   <header>GuiWorkArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
